tests/php/Integration/Api/ApiFacadeTest.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace PhelTest\Integration\Api;
+
+use Gacela\Framework\Bootstrap\GacelaConfig;
+use Gacela\Framework\Gacela;
+use Phel\Api\ApiFacade;
+use PHPUnit\Framework\TestCase;
+
+final class ApiFacadeTest extends TestCase
+{
+    public static function setUpBeforeClass(): void
+    {
+        Gacela::bootstrap(__DIR__, GacelaConfig::withPhpConfigDefault());
+    }
+
+    public function test_number_of_grouped_functions(): void
+    {
+        // This test in isolation works, but when running with all other integration tests fails,
+        // because the IntegrationTest loads already the core and all internal code, which
+        // it crashes when it tries to load all phel funcs again. See: `PhelFnLoader::loadAllPhelFunctions()`
+        $this->markTestSkipped('Useful to debug the facade, but useless to keep it in the CI');
+        $facade = new ApiFacade();
+        $groupedFns = $facade->getNormalizedGroupedFunctions();
+
+        self::assertCount(212, $groupedFns);
```

Comment:
As soon as we create a new function, this number will fail, isn't it?
I would suggest doing any other test for this to avoid updating this test each time we add a new one, it can become a bit tedious, what do you think? I cannot suggest any other possible test at the moment 😄

Reply:
Exactly... That was another reason to skip the whole test. This is a useful test for integration, to verify that the whole thing works isolated, otherwise it's a fragile test that will break whenever we add new functions. 
This might be subjective at this point. I don't have a strong opinion on this at this moment 🤔

Reply:
Another thing: The test looks like it also contains the `phel\core` function. But the aren't in the provided list. Don't know if there is an easy fix for the problem.

Reply:
Actually, there is no need to mention the `phel\core` as these functions are always loaded by default. Maybe I should filter them out if we don't call for them? That might be an interesting idea 🤔